apps/challenges/views.py

```diff
@@ -808,6 +808,47 @@ def create_challenge_using_zip_file(request, challenge_host_team_pk):
                 response_data, status=status.HTTP_400_BAD_REQUEST
             )
 
+        # Move this to another method.
+        if(data.get("submission_meta_attributes")):  # To ensure that the schema for submission metaattributes is valid.
```

Comment:
@KhalidRmb there is another API for validate challenge config we have to add these changes to that  API too. It will be used for github integration of challenge creation flow.